pkg/globalnet/controllers/controllers_suite_test.go

```diff
@@ -148,20 +170,49 @@ func (t *testDriverBase) deletePod(p *corev1.Pod) {
 	Expect(t.pods.Namespace(p.Namespace).Delete(context.TODO(), p.Name, metav1.DeleteOptions{})).To(Succeed())
 }
 
-func getGlobalEgressIPStatus(client dynamic.ResourceInterface, name string) *submarinerv1.GlobalEgressIPStatus {
+func (t *testDriverBase) createService(service *corev1.Service) *corev1.Service {
+	test.CreateResource(t.services, service)
+	return service
+}
+
+func (t *testDriverBase) createServiceExport(s *corev1.Service) {
+	test.CreateResource(t.serviceExports, &mcsv1a1.ServiceExport{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      s.GetName(),
+			Namespace: s.GetNamespace(),
+		},
+	})
+}
+
+func (t *testDriverBase) getGlobalIngressIPStatus(name string) *submarinerv1.GlobalIngressIPStatus {
+	status := &submarinerv1.GlobalIngressIPStatus{}
+	getStatus(t.globalIngressIPs, name, status)
```

Comment:
```suggestion
	if !getStatus(t.globalIngressIPs, name, status) {
	    return nil
	}
```

Reply:
I did this to avoid having to check for nil in all call sites.  If the `status`  field was missing, returning an empty struct is equivalent to nil. Hence I removed the return value from `getStatus`.